core/src/main/java/com/fincity/saas/core/functions/storage/ReadPageStorageObject.java

```diff
@@ -69,6 +79,13 @@ public AppDataService getAppDataService() {
 	@Override
 	public FunctionSignature getSignature() {
 
+	 Schema objectSchema = new Schema()
+    .setType(Type.of(SchemaType.OBJECT))
+    .setProperties(Map.of(
+        "direction",Schema.ofString("direction").setEnums(List.of(new JsonPrimitive("ASC"),new JsonPrimitive("DESC"))).setDefaultValue(new JsonPrimitive("DESC")),
+        "property", Schema.ofString("property").setDefaultValue(new JsonPrimitive("updatedAt"))
```

Comment:
Don't default property, add null handling and ignore case.

Please check 
https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/domain/Sort.Order.html

Reply:
Add setName("SortOrder")